h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -1077,9 +1077,15 @@ static void closeNewChunks(NewChunk[] nchks) {
     fs.blockForPending();
   }
 
+  // Build real Vecs from loose Chunks, and finalize this Frame.  Called once
+  // after any number of [create,close]NewChunks. This method also unlocks the frame.
+  void finalizePartialFrame(long[] espc, String[][] domains, byte[] types) {
+    finalizePartialFrame(espc, domains, types, false);
+  }
+  
   // Build real Vecs from loose Chunks, and finalize this Frame.  Called once
   // after any number of [create,close]NewChunks.
-  void finalizePartialFrame( long[] espc, String[][] domains, byte[] types ) {
+  void finalizePartialFrame(long[] espc, String[][] domains, byte[] types, boolean leaveLocked) {
```

Comment:
The default for unlocking is to unlock, it is IMHO better to have a true (=posive case) as a default. I would thus name it `unlock` with default value `true`.

Reply:
ok, np. will change